server/routes/main.js

```diff
@@ -3,8 +3,37 @@
 const express = require('express');
 const router = express.Router();
 
-router.get('/', (req, res) => {
-  console.log('Hit');
+const User = require('../../database/models/User');
+const Contact = require('../../database/models/Contact');
+
+// this is the /contacts that the user will see, not /api/contacts
+router.get('/contacts', (req, res) => {
+  return Contact.fetchAll()
+    .then(fetched => {
+      return res.json(fetched);
+    });
+});
+
+router.get('/contacts/:id', (req, res) => {
```

Comment:
This also looks like a dup of your get contact route in your contacts route file.